x-pack/plugins/fleet/common/openapi/bundled.yaml

```diff
@@ -2277,10 +2277,16 @@ components:
               - api_keys
               - fleet_admin_user
               - fleet_server
+        missing_optionnal_features:
+          type: array
+          items:
+            type: string
+            enum:
               - encrypted_saved_object_encryption_key_required
       required:
         - isReady
         - missing_requirements
+        - missing_optionnal_features
```

Comment:
would `missing_optional_requirements` be more consistent with the other array?

Reply:
optional requirements it is something correct? Does a requirement should be required?  If it is I will make the change :)